polaris-react/src/components/PageActions/PageActions.scss

```
@@ -2,10 +2,9 @@
 
 .PageActions {
   margin: 0 auto;
-  padding: var(--p-space-5);
-  border-top: var(--p-border-width-1) solid var(--p-color-border-subdued);
+  padding: var(--p-space-4);
```

We should put this behind the beta 

```suggestion
  padding: var(--p-space-5);
  border-top: var(--p-border-width-1) solid var(--p-color-border-subdued);
  
  #{$se23} & {
    padding: var(--p-space-4);
  }
```